SW_Flow_lib.c

```diff
@@ -247,25 +241,24 @@ void litter_intercepted_water(double *pptleft, double *wintlit, double blitter,
 }
 
 /**
-	\fn void infiltrate_water_high(double swc[], double drain[], double *drainout, double pptleft,
-	unsigned int nlyrs, double swcfc[], double swcsat[], double impermeability[],
-	double *standingWater)
-
-	\brief Satured percolation function. Infiltrate water into soil layers under high
-	water conditions.
-
-	\param swc.  An array of doubles. Soilwater content in each layer before drainage.
-	\param swcfc. An array of doubles. Soilwater content in each layer at field capacity.
-	\param swcsat. An array of doubles. Soilwater content in each layer at saturation.
-
-	\param impermeability. An array of doubles. Impermeability measures for each layer.
-	\param pptleft. Daily precipitation available to the soil.
-	\param nlyrs. Number of layers to drain from.
-
-	\return drain An array of doubles. Drainage amount in each layer
-	\return swc. An array of doubles. Soilwater content in each layer after drainage.
-	\return standingWater. A double. Remaining water on surface.
-	\return drainout. A double. Drainage from last layer.
+@brief Infiltrate water into soil layers under high water conditions, otherwise
+      known as saturated percolation.
+
+@param swc  Displays soilwater content in each layer before drainage, (cm^3 H2O).

```

Comment:
Doublecheck: Do these superscripts come up as squared and cubed in the .html version?